examples/blender/blender.py

```diff
@@ -47,10 +48,22 @@ async def worker(ctx: WorkContext, tasks):
             ctx.run("/golem/entrypoints/run-blender.sh")
             output_file = f"output_{frame}.png"
             ctx.download_file(f"/golem/output/out{frame:04d}.png", output_file)
-            yield ctx.commit()
-            # TODO: Check if job results are valid
-            # and reject by: task.reject_task(reason = 'invalid file')
-            task.accept_result(result=output_file)
+            try:
+                # Set timeout for executing the script on the provider.
+                # If the timeout is exceeded, this worker instance will be
+                # shut down and all remaining tasks, including the current one,
+                # will be computed by other providers (hopefully).
+                yield ctx.commit(timeout=timedelta(seconds=120))
```

Comment:
maybe it should be explained that the value itself is completely arbitrary and specific to just this example?

Reply:
Fixed in https://github.com/golemfactory/yapapi/pull/122/commits/6450cbccec84d59d6a1736d99f11bd8d89a7f537